Formula/deck.rb

```diff
@@ -5,29 +5,29 @@
 class Deck < Formula
   desc "Declarative configuration for Kong"
   homepage "https://github.com/kong/deck"
-  version "1.47.0"
+  version "1.47.1"
 
   on_macos do
-    url "https://github.com/Kong/deck/releases/download/v1.47.0/deck_1.47.0_darwin_all.tar.gz"
-    sha256 "103826354f93f1124c07b2c674bfc0b43a2107f57ae2db41cf522e52db027d1f"
+    url "https://github.com/Kong/deck/releases/download/v1.47.1/deck_1.47.1_darwin_all.tar.gz"
+    sha256 "c41f553aaf827b3b63f0c4354462462dff5ea80088a7aee582418f9a34a16bca"
 
     def install
       bin.install "deck"
     end
   end
 
   on_linux do
-    if Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
```

Comment:
@Prashansa-K need a review here - the `if condition` order has changed in the artifact for linux.